components/brave_wallet/browser/keyring_service.cc

```diff
@@ -915,141 +922,104 @@ std::string KeyringService::GetMnemonicForKeyringImpl(
   return std::string(mnemonic->begin(), mnemonic->end());
 }
 
-void KeyringService::AddFilecoinAccount(const std::string& account_name,
-                                        const std::string& network,
-                                        AddAccountCallback callback) {
-  if (!IsFilecoinEnabled()) {
-    std::move(callback).Run(false);
-    return;
-  }
-
-  std::string keyring_id = GetFilecoinKeyringId(network);
-
-  if (!LazilyCreateKeyring(keyring_id)) {
-    VLOG(1) << "Unable to create Filecoin keyring";
-    std::move(callback).Run(false);
-    return;
-  }
-
-  auto* keyring = GetHDKeyringById(keyring_id);
-  if (!keyring) {
-    std::move(callback).Run(false);
-    return;
-  }
-
-  auto account = AddAccountForKeyring(keyring_id, account_name);
-  if (!account) {
-    std::move(callback).Run(false);
-    return;
-  }
-
-  SetSelectedAccountForCoinSilently(*account);
-  SetSelectedCoin(profile_prefs_, mojom::CoinType::FIL);
-
-  NotifyAccountsChanged();
-  NotifyAccountsAdded(*account);
-
-  std::move(callback).Run(keyring);
-}
```

Comment:
KeyringService::AddAccount now accepts target keyring, so this method is no longer needed